layout-normalizer/src/gpos/pairpos.rs

```diff
@@ -233,7 +233,7 @@ mod tests {
         }
 
         fn build_exactly_one_subtable(self) -> PairPos {
-            let mut varstore = VariationStoreBuilder::new();
+            let mut varstore = VariationStoreBuilder::new(0);
```

Comment:
how about we add a `VariationStoreBuilder::nop()` alternative constructor that would set axis_count = 0? It doesn't make much sense for the _actual_ `new()` default constructor to take an axis_count that is equal to 0, because there's nothing to "build" in that case, the varstore won't contain any regions or deltas if there are no axes...

Reply:
given that this should only ever be needed in tests I'm not too fussed either way? That we need this at all is mostly just indicates there's something smelly in our API (although there's a rationale for that)